walkthroughs/week-5-tdd/project/src/test/java/com/google/sps/EventTest.java

```diff
@@ -34,20 +34,19 @@
   private static final int TIME_0830 = TimeRange.getTimeInMinutes(8, 30);
   private static final int TIME_1330 = TimeRange.getTimeInMinutes(13, 30);
 
-  private static final Event EVENT_A = new Event(
-      "Event A", TimeRange.fromStartDuration(TIME_0830, DURATION_30_MINUTES), Arrays.asList());
-  private static final Event EVENT_B = new Event(
-      "Event B", TimeRange.fromStartDuration(TIME_1330, DURATION_60_MINUTES), Arrays.asList());
-
   @Test
-  public void orderEventByStartTime() {
-    Event[] expected = {EVENT_A, EVENT_B};
-
-    PriorityQueue<Event> eventQueue = new PriorityQueue<>(Event.ORDER_BY_START_TIME);
-    eventQueue.add(EVENT_B);
-    eventQueue.add(EVENT_A);
-    Event[] actual = eventQueue.toArray(new Event[eventQueue.size()]);
-
+  public void orderEventsByStartTime() {
+    // The late event should be added to the array before the early event. This means sorting with
+    // the ORDER_BY_START_TIME comparator should re-order them
+    Event earlyEvent = new Event("Early Event",
+        TimeRange.fromStartDuration(TIME_0830, DURATION_30_MINUTES), Arrays.asList());
+    Event lateEvent = new Event(
+        "Late Event", TimeRange.fromStartDuration(TIME_1330, DURATION_60_MINUTES), Arrays.asList());
+
+    Event[] actual = {lateEvent, earlyEvent};
+    Arrays.sort(actual, Event.ORDER_BY_START_TIME);
```

Comment:
I'm sorry, I didn't really think carefully about your test plan here. Since you're really just trying to test the comparator, you should just write tests for the comparator! This test also implicitly tests the `Arrays.sort(...)` method. Unit tests should be as focused as possible; exercising extra logic gives more opportunities for flaws to enter your testing logic (as we saw previously with the priority queue).

If I were writing tests for this comparator, I'd initially start with three separate tests: one for which `Event.ORDER_BY_START_TIME.compare(...)` should return negative, one 0, and one positive. You might be able to think of more.

This isn't necessary to address in this PR (or even at all; there will be more opportunities to write tests for the capstone).